### PR TITLE
Document CNV binding method webui option

### PR DESCRIPTION
**Pull request by booxter**

### CODE REVIEW

**Comment by booxter** [comment]
@SchSeba this is similar to https://github.com/kubevirt/user-guide/pull/236/ except that I dropped mentioning `sriov` because it's not included into CNV 2.0

**Review by SchSeba — APPROVED** [review]

File: modules/cnv_networking_wizard_fields_web.adoc [hunk 1]
```diff
@@ -19,6 +19,9 @@
 |NETWORK CONFIGURATION
 |List of available *NetworkAttachmentDefinition* objects.
 
+|BINDING METHOD
+|List of available binding methods. For pod networking, the possible values are `masquerade` and `bridge`. The `masquerade` method is not supported for non-default networks.
```
Comment (dankenigsberg) [inline modules/cnv_networking_wizard_fields_web.adoc]: It is very important **not** to allow `bridge` on the default network. It is spells only trouble.

For the default pod network, the only possible value is `masquerade`. For secondary networks the only possible method is `bridge`.

**Review by dankenigsberg — CHANGES_REQUESTED** [review]

**Comment by booxter** [comment]
/hold

This will need to be reproposed for enterprise-4.1 branch when the new layout for CNV docs is backported there.

**Comment by ousleyp** [comment]
Hey @booxter do you mind closing this PR? The binding method info has already been merged:

https://github.com/openshift/openshift-docs/pull/14351/files#diff-f5fef2dd1faeebdba55e9b5ab92c1961

Thanks! And I'm sorry for all the confusion - when we're done modularizing the docs, we'll send out an email with new (and improved) instructions for dev contributions. Thanks for your patience.

**Pull request closed without merging**